tests/ses/nodes_preparation.pm

```diff
@@ -22,10 +22,10 @@ sub run {
     select_console 'root-console';
     # configure and start ntp, ntp server for nodes is master
     if (check_var('HOSTNAME', 'master')) {
-        my $num_nodes = get_var('NODE_COUNT');
-        barrier_create('network_configured', $num_nodes + 1);
-        barrier_create('deployment_done',    $num_nodes + 1);
-        barrier_create('all_tests_done',     $num_nodes + 1);
+        my $all_ses_nodes = get_var('NODE_COUNT') + 1;
```

Comment:
Another option (feel free to ignore) is to calculate cluster size:
```
my $children = get_children;
my $nodes    = (keys %$children);
barrier_create('network_configured', $nodes);
```

Reply:
Would have to use mmapi, maybe later, will leave it for now.